mugoh/qstioner
# #162977962 Create an index page for the UI

#### What does this PR do?
Create the landing page for the site.

#### Description of Task to be completed?
The site should have a functional index page. A visitor can know the services offered on landing on this page.

#### How should this be manually tested?
1. Clone the repo `git clone https://github.com/hogum/qstioner.git`
2. Check into the root directory of the repo `cd qstioner`
3. Open the index file using aa browser of your choice
- 
 ``` shell 
$ name-of-your-broswer index.html
```
or
- Open the repo directory manually and double-click on the index.html file. This will open the page in your default browser.

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#162977962](https://www.pivotaltracker.com/story/show/162977962)

#### Screenshots
![screenshot from 2019-01-05 13-41-12](https://user-images.githubusercontent.com/44059971/50723607-21199c00-10f1-11e9-92b1-d1f97e7b5da5.png)
